operations/accounts.md

```diff
@@ -0,0 +1,64 @@
+# Accounts
```

Comment:
SUMMARY.md will need updating to add the new page accounts.md to the Table of Contents.